src/platform/qt/Window.cpp

```diff
@@ -1429,6 +1429,20 @@ void Window::setupMenu(QMenuBar* menubar) {
 	}
 	m_config->updateOption("fastForwardRatio");
 
+	addGameAction(tr("Fast forward speed increment"), "fastForwardUp", [this] {
+		float oldRatio = 0.0;
+		oldRatio = m_config->getOption("fastForwardRatio", oldRatio).toFloat();
+		float newRatio = oldRatio + 1.0;
+		m_config->setOption("fastForwardRatio", QVariant(newRatio));
+	}, "emu");
+
+	addGameAction(tr("Fast forward speed decrement"), "fastForwardDown", [this] {
+		float oldRatio = 0.0;
+		oldRatio = m_config->getOption("fastForwardRatio", oldRatio).toFloat();
+		float newRatio = oldRatio - 1.0;
+		m_config->setOption("fastForwardRatio", QVariant(newRatio));
```

Comment:
Ditto, but also you need to clamp this to be a `2.0f` minimum.

Reply:
Changed it to only use one variable, and added the clamp.